engine/src/cmd/base_interface.cpp

```diff
@@ -1238,7 +1238,7 @@ void BaseInterface::Room::Launch::Click( BaseInterface *base, float x, float y,
                 playa->getAIState()->Communicate( c );
             abletodock( 5 );
             if (playa->name == "return_to_cockpit")
-                if (playa->faction == playa->faction)
+                //if (playa->faction == playa->faction) // This will always return true :/
```

Comment:
perhaps this was suppose to be `playa->faction == bas->faction`?

Reply:
Changed with a note, for history's sake